Stratis.Bitcoin.Features.Wallet/IWalletManager.cs

```diff
@@ -210,6 +210,12 @@ public interface IWalletManager : IDisposable
         /// <returns>The wallet filenames, along with the folder in which they're contained.</returns>
         (string folderPath, IEnumerable<string>) GetWalletsFiles();
 
+        /// <summary>
```

Comment:
I would rather have AreWallets be called ContainsWallets or something like that.

Reply:
Yes I like that better. I changed it to ContainsWallets.